css/style.css

```diff
@@ -0,0 +1,1085 @@
+    margin: 0 4px;
+  }
+
+  .headline {
+    font-family: "Russo One", sans-serif;
+    font-size: 150%;
+    text-align: justify;
+    padding-top: 225px;
+  }
+
+  #br {
+    display: none;
+  }
+
+  #spn {
+    display: block;
+  }
+
+  .side-headline {
+    font-family: "Russo One", sans-serif;
+    font-size: 100%;
+    padding-top: 125px;
+    text-align: left;
+  }
+
+  .side-pic-content i {
+    margin: 0 4px;
+  }
+
+  /* styling latest news section */
+  .section-4 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    margin: auto;
+    grid-gap: 5px;
+    width: 90%;
+  }
+
+  /* styling latest news section */
+
+  .newscont {
+    width: 80%;
+    margin: auto;
+  }
+
+  .newsitem {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 10px;
+    justify-content: center;
+  }
+
+  .latest-news {
+    margin-top: 150px;
+  }
+
+  .item a {
+    text-decoration: none;
+    color: black;
+    font-size: 16px;
+    font-family: "Varela Round", sans-serif;
+    font-weight: bold;
+  }
+
+  .author {
+    font-family: "Varela Round", sans-serif;
+    font-size: 13px;
+    color: grey;
+  }
+
+  .title a {
+    font-family: "Russo One", sans-serif;
+    text-decoration: none;
+    color: #f42;
+    font-size: 28px;
+    font-weight: bolder;
+  }
+
+  .footnav {
+    width: 35%;
+    margin: 2% auto;
+    font-size: 75%;
+    text-align: center;
+    padding-bottom: 20px;
+  }
+}
+
+/* responsiveness for mobile devices */
+@media only screen and (max-width: 767px) {
+  /* navigation small */
+
+  .navtop {
+    display: none;
+  }
+
+  /* navlogom&s style */
+  .navlogom-s {
+    display: block;
+    width: 85px;
+    height: 35px;
+    padding-top: 15px;
+    padding-left: 20px;
+  }
+
+  .navlogom-s img {
+    width: 70%;
+  }
+
+  .navli2 {
+    display: none;
+  }
+
+  /* navigation fixed styling */
+  .header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 100;
+  }
+
+  .navfixed {
+    height: 50px;
+    font-family: "Nova Square", cursive;
+    background-color: white;
+    font-size: 90%;
+  }
+
+  #searchbtn {
+    border-bottom: 1px rgba(84, 84, 84, 0.27) solid;
+    border-top: 1px rgba(84, 84, 84, 0.27) solid;
+    padding-bottom: 10px;
+    padding-left: 15px;
+    padding-top: 15px;
+    width: 35px;
+    font-size: 110%;
+  }
+
+  #hamburger {
+    display: block;
+    color: orangered;
+    margin-left: 10px;
+    font-size: 120%;
+  }
+
+  .hiddenmd {
+    display: none;
+  }
+
+  /* yellow nav styling */
+
+  .yellow-nav {
+    background-color: #ffe712;
+    height: 30px;
+    padding-top: 20px;
+    text-align: center;
+    clear: both;
+  }
+
+  /* main content */
+  .main-news-section {
+    display: grid;
+    grid-gap: 6px;
+    grid-template-areas:
+      "main"
+      "side-top"
+      "side-bottom";
+    margin: 4px 4px 5% 4px;
+  }
+
+  .main-pic {
+    /* background:  linear-gradient(rgba(196, 32, 32, 0.25)),url("./images/main-pic-big.png"); */
```

Comment:
Please remove this commented piece of code.

Reply:
Commented code removed.